web/src/containers/Routes/index.tsx

```diff
@@ -11,10 +11,28 @@ import { withScrollRestoration } from "../../utility/withScrollRestoration";
 import { ReactNode } from "react";
 import { useRecoilValue } from "recoil";
 
+const showGemsOnly = false;
```

Comment:
this should definitely be moved into the config and then referenced dynamically

Reply:
I'm no frontender so no clue where to put it
Left it there as a place holder

Reply:
You can mimic the code in my PR if you want for the config stuff, and then I'll close it.